pangea-sdk/service/audit/api.go

```diff
@@ -634,7 +633,7 @@ func (ee *EventEnvelope) VerifySignature() EventVerification {
 		return Failed
 	}
 
-	v, err := signer.NewVerifierFromPubKey(publicKey)
+	v, _ := signer.NewVerifierFromPubKey(publicKey)
```

Comment:
We'll definitely refactor this later, I'm not sure if masking the errors is ideal.

Reply:
By all means, only reason I included it was because the linter I was using noted that that particular value of `err` was never actually used. I actually updated this to instead use the `err` on return by convention.